.github/workflows/lock.yml

```diff
@@ -0,0 +1,27 @@
+name: "Lock outdated threads"
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
This is scheduled to run once a day at the top of the hour.

https://crontab.guru/#0_0_*_*_*

Reply:
TIL about crontab.guru

Reply:
Neat-o!